apps/graphql/src/dataloaders/ItinerariesloaderTypes.js

```diff
@@ -30,68 +30,71 @@ export type RouteItemType = {|
 // start - new structure
 
 export type CountryType = {|
-  +id: string,
-  +name: string,
-  +code: string,
-  +slug: string,
-  +flagURL: string,
+  +id: ?string,
+  +name: ?string,
+  +code: ?string,
+  +slug: ?string,
+  +flagURL: ?string,
 |};
 
-// @TODO rename PriceType
 export type PriceType = {|
-  +amount: number,
-  +currency: string,
+  +amount: ?number,
+  +currency: ?string,
 |};
 
 export type LocationType = {|
   +id: string,
-  +locationId: string,
-  +name: string,
-  +timezone: string,
-  +country: CountryType,
+  +locationId: ?string,
+  +name: ?string,
+  +timezone: ?string,
+  +country: ?CountryType,
 |};
 export type DateType = {|
-  +local: Date,
-  +utc: Date,
+  +local: ?Date,
+  +utc: ?Date,
 |};
-export type ProviderType = {|
-  +id: string,
-  +name: string,
+
+export type VehicleTypeType = 'bus' | 'train' | 'aircraft';
```

Comment:
Same as below, we can relay on API response.

Reply:
Done